content/en/service_management/workflows/connections.md

```diff
@@ -105,6 +106,71 @@ To learn how to restrict connection use, see [Access and Authentication][4].
+
+You can create groups of connections so that your workflows can authenticate into the correct account or accounts based on the given inputs. Connections can be grouped together only if they share the same integration (for example, you cannot group GCP and AWS connections within the same group). 
+
+You define the members of a connection group using a connection's _Identifier Tags_. For example, you can create a connection group consisting of AWS accounts that have the `account_id` tag.
+
+Each connection in the group must have a set of unique identifier tags so that a workflow can dynamically select the correct connection at runtime. For example:
+- `connectionA {account_id:123456789}` and `connectionB {account_id:987654321}` can be grouped together.
+- `connectionA {account_id:123456789}` and `connectionC {account_id:123456789}` can't be grouped, because the group would contain duplicate tag values.
+
+### Create a connection group
+
+<div class="alert alert-info"><strong>Note</strong>: You can only add connections to a group if you have <a href="/service_management/workflows/access/#restrict-access-on-a-specific-connection">Resolver permission</a> for them.</div>
+
+To create a connection group:
+
+1. Navigate to the [connections list][3].
+1. On the left, click **Groups**.
+1. Click **+ New Group**, then select an integration.
+1. Enter a group name, then enter a set of up to three **Identifier Tags** that the connections you want to include in your group all have.
+1. Under **Confirm Group**, use the checkboxes to select the specific members of your group.
+1. Click **Next, Confirm Access**, then choose your desired access level for the group.
+1. Click **Create**.
+
+### Use a connection group
+
+To use a connection group:
+
+1. In your workflow, select an action that requires a connection.
+1. In the **Connection** field, in the drop-down, select the desired connection group under **Groups**.
+1. Fill in the desired values for the connection group **Identifiers**. For example, if your connection group is defined using the `env` Identifier Tag, and you have two environments, `prod` and `staging`, you could use either of those values (or an expression that evaluates to one of those values).
+1. Fill in any other required step values, then click **Save** to save your workflow.
+
+**Note**: You can only use connections within a group if you have [Resolver permission][12] for those connections. If a workflow tries to use a connection you don't have Resolver permission for, the workflow fails with a `403 Forbidden` error. To fix this issue, you can:
+- Configure your workflow so that it can't point to a connection you don't have Resolver permission for.
+- Remove the connection you don't have Resolver permission for from the connection group.<div class="alert alert-danger"><strong>Note</strong>: If you are using a connection group for multiple workflows, removing a connection that another workflow relies on causes that workflow to fail.</div>
```

Comment:
```suggestion
- Remove the connection that doesn't have Resolver permission for the connection group.<div class="alert alert-danger"><strong>Note</strong>: If you are using a connection group for multiple workflows, removing a connection that another workflow relies on causes that workflow to fail.</div>
```
Does this still work? I stumbled over "for from" and am trying to revise it to avoid that, but am not sure I captured the intended meaning.

Reply:
Note that in the preview, this alert doesn't render with the correct font size -- I don't think we're expected to include an alert in steps, since usually it goes at the top of a section (even if I like your approach better since the user will have better context where you included it). It's a very minor difference and I don't think it needs to be a stopship concern at all, just flagging in case that matters to you.

Reply:
Yeah, I noticed the font size thing.

I think the meaning changes if you remove the word, but how about...